app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+
+    @comment.save
```

Comment:
same here - what happens if a user enters a blank comment and the comment doesn't save?